### PR TITLE
Add test ignored case to the logger

### PR DESCRIPTION
**Pull request by gricsi**

### CODE REVIEW

File: lib/team_city_formatter/formatter.rb [hunk 1]
```diff
@@ -104,20 +108,24 @@ def before_scenario_outline(cuke_scenario_outline)
...
-            example_row.send(:data).values
-          end.map do |example_column_values|
-            Example.new.tap do |x|
-              x.column_values = example_column_values
+            cuke_example_rows.map do |example_row|
+              example_row.send(:data).values
+            end.map do |example_column_values|
+              Example.new.tap do |x|
+                x.column_values = example_column_values
+              end
```
Comment (kevinrood) [inline lib/team_city_formatter/formatter.rb]: @gricsi it looks like these lines were simply indented another couple of spaces. Could you please revert the change here.

Comment (gricsi) [inline lib/team_city_formatter/formatter.rb]: @kevinrood yes I can.

File: lib/team_city_formatter/logger.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def test_suite_finished(test_suite_name)
     end
 
     def test_started(test_name)
-      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' timestamp='#{timestamp}' captureStandardOutput='true']")
+      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' captureStandardOutput='true' timestamp='#{timestamp}']")
```
Comment (kevinrood) [inline lib/team_city_formatter/logger.rb]: @gricsi is there a specific reason you moved the `captureStandardOutput` attribute here and in `#test_failed`?

File: lib/team_city_formatter/logger.rb [hunk 3]
```diff
@@ -38,11 +42,13 @@ def render_output(text)
...
-        .gsub(']', '|]')
-        .gsub("\r", '|r')
-        .gsub("\n", '|n')
+          .gsub(':', ' -')
+          .gsub('|', '||')
+          .gsub("'", "|'")
+          .gsub(']', '|]')
+          .gsub('[', '|[')
+          .gsub("\r", '|r')
+          .gsub("\n", '|n')
```
Comment (kevinrood) [inline lib/team_city_formatter/logger.rb:51]: @gricsi thanks for the additions here :smile:

File: lib/team_city_formatter/formatter.rb (hunk 1 above)
Comment (kevinrood) [inline lib/team_city_formatter/formatter.rb]: thank you @gricsi

**Comment by kevinrood** [comment]
@gricsi one additional favor I would ask is, I have not merged #5, but I really should merge that change before yours as the pull request has been open for some time. Would you mind rebasing your changes in this pull request over master after I merge #5?

File: lib/team_city_formatter/logger.rb (hunk 3 above)
Comment (gricsi) [inline lib/team_city_formatter/logger.rb:51]: @kevinrood unfortunately ':' also makes some problems on the TC report. :)

File: lib/team_city_formatter/logger.rb (hunk 2 above)
Comment (gricsi) [inline lib/team_city_formatter/logger.rb]: @kevinrood I just saw in the documentation and everywhere the timestamp was in the end. It seems more consistent for me but I can revert this line if you want.

Comment (kevinrood) [inline lib/team_city_formatter/logger.rb]: @gricsi you can leave it at the end. Could you please rebase over the mater branch? I just merged a pull which resolved a problem when running cucumber test suites with this output formatter in parallel.

**Comment by gricsi** [comment]
@kevinrood done :)

**Comment by kevinrood** [comment]
@gricsi perfect - thank you!

Would you be in a position to run the version in the `master` branch with your test suite to make sure everything is working?

**Comment by gricsi** [comment]
How can I update the gem to use this master version? I made it and tried it my changes on our test sets before I created the PR.

**Comment by kevinrood** [comment]
@gricsi if you are using `bundler`, please see [this reference](http://bundler.io/git.html). It should be as simple as adding this to your `Gemfile`:

``` ruby
gem 'teamcity_formatter', :git => 'https://github.com/kevinrood/teamcity_formatter.git', :branch => 'master'
```

Then: `bundle install`

**Comment by gricsi** [comment]
@kevinrood It's working fine for my test suite.

**Comment by kevinrood** [comment]
@gricsi this pull request is now available in version 0.8.2 of the gem, which was just published :smile:

**Comment by gricsi** [comment]
@kevinrood thanks a lot :)
